master/getting-started/docker/index.md

```diff
@@ -1,4 +1,47 @@
 ---
 title: Calico with Docker
 ---
-More information coming soon!
+
+Calico implements a Docker network plugin that can be used to provide routing and advanced network policy for Docker containers.
+
+Use the navigation bar on the left to view information on Calico for Docker,
+or continue reading for an overview of recommended guides to get started.
+
+
+## Installation
+
+#### [Requirements](installation/requirements)
+
+Information on running etcd and configuring Docker for multi-host networking.
```

Comment:
Yes true, but I would see those guides being elsewhere and referenced from the manual install guide - and so therefore those common requirements guides won't be here....